### PR TITLE
Hello Microverse project

### PR DESCRIPTION
**Pull request by 3nityGitHub**
# In this PR,

- I added the index.html file
- I added style.css file
- I added Hello microverse message to the index.html file
- I added a class called header and styled it in the style.css file
- I also added linters.yml file to check for errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 Boilerplate</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+	<h1>Hello Microverse!</h1>
+  </body>
```
Comment (HAMMAS-SALEEM) [inline index.html]: - [ ] As the previous reviewer asked you to add class to your `h1` tag as shown below :arrow_down:
`<h1 class="header">Hello Microverse</h1>`

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (HAMMAS-SALEEM) [inline readme.md:43]: - [ ] You have added a license link here but you forgot to add the license file  :fearful: No worries you can add it using these [guildlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f574770523531743971623739534e464f74732f67697068792e676966

## Hey @3nityGitHub :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Descriptive PR Title

Good job on implementing some of the requested changes, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
Hi @3nityGitHub , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**